lib/eex/lib/eex/tokenizer.ex

```diff
@@ -101,19 +102,19 @@ defmodule EEx.Tokenizer do
 
   # Tokenize an expression until we find %>
 
-  defp expr([?%, ?> | t], line, column, buffer) do
+  defp expr([?%, ?> | t], line, column, _opts, buffer) do
     {:ok, buffer, line, column + 2, t}
   end
 
-  defp expr('\n' ++ t, line, _column, buffer) do
-    expr(t, line + 1, 1, [?\n | buffer])
+  defp expr('\n' ++ t, line, _column, opts, buffer) do
+    expr(t, line + 1, opts[:column_padding] + 1, opts, [?\n | buffer])
```

Comment:
Wrong name, which means this isn't tested at all? 🤔

Reply:
good catch, it broke the build too. Fixed!